### PR TITLE
Bug fix for notebooks

### PR DESCRIPTION
**Pull request by huard**
Fixed custom output command not rendering correctly
Convert config and output paths to absolute paths

### CODE REVIEW

**Review by Zeitsperre — APPROVED** [review]
Looks good

**Comment by tlvu** [comment]
Is this PR compatible with the current Jupyter env and can be merged immediately or this PR is only for the upcoming Jupyter env and have to wait for that one to go live before this PR can be merged?

If this is for the upcoming Jupyter env only, can this PR be merged to the branch `fixnbs_pavics_betaimage` (https://github.com/CSHS-CWRA/RavenPy/pull/356) from Travis so I can test all upcoming changes together?

**Comment by huard** [comment]
It's a bugfix that I believe is compatible with the current env.

**Pull request merged**
